MetaMask/metamask-extension
# Use @redux-devtools/remote, update README

## Explanation



The reason for this PR is that new contributors were unable to install `remotedev-server`. Also the package has been deprecated.

This PR:
1. Replaces the deprecated `remotedev-server` with the official `@redux-devtools/cli`
2. Replaces the deprecated `remote-redux-devtools` with the official `@redux-devtools/remote`

## Manual Testing Steps



1. Start the dev server: `yarn start`
2. Start Redux devtools: `yarn devtools:redux`
3. Open the MM extension
4. Right click > Redux Devtools > Remote Devtools
5. Now you should be able to see the Redux store and history of actions

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
